spec/lib/jobs/obm-control-spec.js

```diff
@@ -151,6 +182,19 @@ describe("Job.Obm.Node", function () {
                 expect(ObmService.prototype.reboot).to.have.been.calledOnce;
             });
         });
+
+        it('should fail to run OBM command if no target or nodeId was specified', function() {
+
+            // local options with no nodeId field
+            var options = {
+                action: 'reboot',
+                obmServiceName: 'ipmi-obm-service'
+            };
+
+            expect(function() {
+                return new Job(options, {}, uuid.v4());
+            }).to.throw('nodeId (string) is required');
```

Comment:
It would be better to expect an `AssertionError` not the specified error message, like `expect(xxx).to.throw(AssertionError)`, so this leave the flexibility to change error message without modifying the unit-test.